### PR TITLE
ELE-4208 Hook up instana to error logging

### PR DESCRIPTION
**Pull request by maddocash**
#### Linked Github ticket
https://github.com/talis/elevate-app/issues/4208

#### Linked Instana docs
https://www.instana.com/docs/website_monitoring/api/#error-tracking

#### Change

- forward errors to instana

### CODE REVIEW

File: js/logging.js
Comment (pogotc) [inline js/logging.js:58]: It's all a bit academic as we're the only ones who use this library, but I still wonder if these changes should be in addition to NewRelic support instead of as a replacement

**Review by pogotc — APPROVED** [review]
Looks good to me, I do wonder if this should be adding support for Instana rather than replacing NewRelic with it, but I don't feel strongly enough about it to not approve the changes.

Only other thing to remember before merging is to bump the version number.

File: js/logging.js
Comment (maddocash) [reply js/logging.js:58]: I think as it's just us and we hopefully wont be using it be the end of the year, I'm going to leave it as is :)

**Pull request merged**
